base/utils/git.zsh

```diff
@@ -90,7 +90,7 @@ __zplug::utils::git::checkout()
     tags[dir]="$(__zplug::core::core::run_interfaces 'dir' "$repo")"
     tags[from]="$(__zplug::core::core::run_interfaces 'from' "$repo")"
 
-    do_not_checkout=( "gh-r" )
+    do_not_checkout=( "gh-r" "prezto" )
```

Comment:
Note this is a workaround and we assume that the user wants the `master` branch of prezto. See #303